lib/core/assertion.js

```diff
@@ -214,6 +214,11 @@ module.exports = new (function() {
         }).replace(/:/g, '').replace(/\s/g, '-').replace(/-\(.+?\)/, '');
         var fileName = path.resolve(path.join(client.options.screenshots.path, 'FAIL_' + message + theBrowser + dateStamp + '.png'));
         client.api.saveScreenshot(fileName, function(result) {
```

Comment:
No. We're not interested in it here.
We're only interested in the screenshot on disk afterwards